dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx

```diff
@@ -207,52 +217,44 @@ const styles = {
 		color: ${textColor};
 		${limitHeight ? 'max-height: 70vh;' : ''}
 		overflow: auto;
-
-		* {
-			box-sizing: border-box;
-		}
-
-		${from.tablet} {
-			border-top: 1px solid ${neutral[0]};
-		}
-
-		b,
-		strong {
-			font-weight: bold;
-		}
 	`,
 	containerOverrides: css`
 		display: flex;
 		flex-direction: column;
 		position: relative;
-		padding: 0 10px;
+		${from.desktop} {
+			padding: 0;
+		}
+		overflow: auto;
+
+		${from.mobile} {
+			padding: 10px 10px 0 16px;
+		}
 
 		${from.tablet} {
 			display: grid;
-			grid-template-columns: 1fr 280px;
+			grid-template-columns: 1fr 1fr 1fr;
```

Comment:
If this isn't set like this the columns on either side collapse and the content which isn't correct. I think a chat with design would be helpful to get the widths of those columns.